src/plaid/plaid.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from '@nestjs/common'
+import { ModelId } from '../core/model'
+import { EncryptionService } from '../encryption/encryption.service'
+import { PlaidItem, tableName } from './plaidItem.entity'
+import { InjectRepository } from '@nestjs/typeorm'
+import { Repository } from 'typeorm'
+import plaid, { Account, Client } from 'plaid'
+import { ConfigService } from '@nestjs/config'
+import { IPlaidConfig } from '../config/plaidConfig'
+
+@Injectable()
+export class PlaidService {
+  private readonly client: Client
+  constructor(
+    private readonly encryptionService: EncryptionService,
+    private readonly configService: ConfigService,
+    @InjectRepository(PlaidItem) private readonly plaidSessionRepo: Repository<PlaidItem>,
+  ) {
+    const config = this.configService.get<IPlaidConfig>('plaid')
+    if (!config) {
```

Comment:
as below, i prefer this as a one liner, but im sure there are people that would disagree

Reply:
Another option could be using the `getOrThrow` utility function I defined and have used elsewhere.

Reply:
Thanks!  Updated to use `getOrThrow`